testdata/test_binary.cc

```diff
@@ -282,16 +282,48 @@ int i = 2;
 
 }  // namespace ns
 
-class Foo {
+static void TestQualifiedId() {
+  // BREAK(TestQualifiedId)
+}
+
+namespace outer {
+
+namespace inner {
+
+class Vars {
  public:
-  static const int x = 42;
-  static const int y;
+  inline static double inline_static = 1.5;
+  static constexpr int static_constexpr = 2;
+  static const unsigned int static_const;
 };
 
-const int Foo::y = 42;
+const unsigned int Vars::static_const = 3;
 
-static void TestQualifiedId() {
-  // BREAK(TestQualifiedId)
+}  // namespace inner
+
```

Comment:
I would also suggest adding anonymous namespaces as a testcase.

Reply:
Anonymous namespaces are actually not supported right now. Variables (and types) declared in anonymous namespaces are unique for compilation units, so it's not clear how to do a proper lookup. Potentially we could try to find the variable from the current CU, but that might be expensive.